tests/ut/codegen/test_pto_codegen_ops.py

```
@@ -1051,5 +1051,78 @@ def main(
+        single = ir.Program([funcs[0]], funcs[0].name, optimized.span)
+        return codegen_instance.generate(single)
+
+    def test_col_sum_codegen(self):
+        """tile.col_sum emits pto.tcolsum with isBinary attribute."""
+
+        @pl.program
+        class Prog:
+            @pl.function(type=pl.FunctionType.InCore)
+            def main(
+                self,
+                input: pl.Tensor[[16, 16], pl.FP32],
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)